src/status_im/ens/core.cljs

```diff
@@ -47,7 +48,8 @@
 
 (defn- on-resolve [registry custom-domain? username address public-key s]
   (cond
-    (= (ethereum/normalized-address address) (ethereum/normalized-address s))
+    (= (-> address ethereum/normalized-address eip55/address->checksum)
```

Comment:
maybe we could add normalization into eip55

Reply:
indeed.. I was thinking of why we'd have two functions to reach the same end goal
I will add normalization into eip55

Reply:
I included normalization in eip55

I left `normalized-address` as it is, as it is used in lots of different places. Later its use maybe replaced by `eip55/address->checksum` in a separate PR